instance/models/instance.py

```diff
@@ -207,52 +205,61 @@ def _sort_log_entries(server_logs, instance_logs):
         next_server_log_entry = partial(next, server_logs, None)
         next_instance_log_entry = partial(next, instance_logs, None)
 
-        log = []
         instance_log_entry = next_instance_log_entry()
         server_log_entry = next_server_log_entry()
 
         while instance_log_entry is not None and server_log_entry is not None:
             if server_log_entry.created < instance_log_entry.created:
-                log.append(server_log_entry)
+                yield server_log_entry
                 server_log_entry = next_server_log_entry()
             else:
-                log.append(instance_log_entry)
+                yield instance_log_entry
                 instance_log_entry = next_instance_log_entry()
 
         while instance_log_entry is not None:
-            log.append(instance_log_entry)
+            yield instance_log_entry
             instance_log_entry = next_instance_log_entry()
 
         while server_log_entry is not None:
-            log.append(server_log_entry)
+            yield server_log_entry
             server_log_entry = next_server_log_entry()
```

Comment:
Out of scope: Are we using Python 3.5?  If we do, the whole function above should be deleted, and we should use `heapa.merge()` instead.  (In Python 3.4, this function doesn't have a `key` parameter, but it might be worthwhile to rewrite the function anyway.)